build.sh

```diff
@@ -44,6 +44,9 @@ install_rpms() {
 
     frozendeps=""
 
+    # We freeze the version for now since we're carrying patches.
+    frozendeps+=" $(echo osbuild{,-ostree,-selinux,-tools}-100-1.fc39.noarch)"
+
```

Comment:
I did something like this over in https://github.com/coreos/coreos-assembler/pull/3675 but since there was `python3-osbuild-100-1.fc39.noarch` I couldn't briefly find a good way to express that in this `echo osbuild` form. I guess maybe the `python3-osbuild` package is tightly bound to the others so it shouldn't matter?